sky/shell/platform/mojo/platform_view_mojo.cc

```diff
@@ -92,6 +95,12 @@ void PlatformViewMojo::Init(mojo::Shell* shell) {
 void PlatformViewMojo::Run(const mojo::String& url,
                            ServicesDataPtr services,
                            mojo::asset_bundle::AssetBundlePtr bundle) {
+
+  mojo::ServiceProviderPtr sp;
+  service_provider_.Bind(mojo::GetProxy(&sp));
+  service_provider_.AddService<::keyboard::KeyboardService>(this);
```

Comment:
`s/::keyboard::KeyboardService/keyboard::KeyboardService/`

We usually skip the leading `::` unless it's needed by the compiler.